### ISSUE TITLE
List of NMR Data

### ISSUE THREAD

**Issue by kyleabeauchamp** [choderalab/open-forcefield-group#2]
I'm going to make each data class as either "test" or "train", which indicates whether I think the systematic error can be sufficiently controlled for quantitative analysis.  Things that are quantitative can be used in the training set, while things that are more qualitative could still be useful in the test set.  Another reason for putting things in the test set is limited information content--an observable could just be mostly useless.
1.  Scalar Couplings of peptides (Train)
2.  Chemical shifts of full proteins (Test)
3.  NOE (Test)
4.  RDC

Regarding (1), here are some references with expt data:

http://pubs.acs.org/doi/abs/10.1021/ja0660406
http://onlinelibrary.wiley.com/doi/10.1002/prot.24000/full
http://www.pnas.org/content/108/5/1794.short

Regarding (2), here are some references:
http://onlinelibrary.wiley.com/doi/10.1002/ange.201001898/full
http://pubs.acs.org/doi/abs/10.1021/ct200094b

Regarding (3), it's mostly a research level project that I'm playing around with, but it's not in production stages yet.
